services/QuillLMS/client/app/bundles/Staff/components/evidence/configureSettings/activityForm.tsx

```diff
@@ -146,6 +166,30 @@ const ActivityForm = ({ activity, handleClickArchiveActivity, requestErrors, sub
   const maxAttemptStyle = activityMaxFeedback.length && activityMaxFeedback !== '<br/>' ? 'has-text' : '';
   const imageAttributionGuideLink = 'https://www.notion.so/quill/Activity-Images-9bc3993400da46a6af445a8a0d2d9d3f#11e9a01b071e41bc954e1182d56e93e8';
 
+  function renderInvalidHighlightLinks(invalidHighlights){
+    const formattedRows = invalidHighlights && invalidHighlights.length && invalidHighlights.map((highlight: InvalidHighlightProps) => {
+      const { rule_id, rule_type  } = highlight;
+      const ruleTypePart = RULE_TYPE_TO_ROUTE_PART[rule_type]
+      const ruleName = RULE_TYPE_TO_NAME[rule_type]
+      const invalidHighlightLink = (<Link to={`/activities/${id}/${ruleTypePart}/${rule_id}`}>{ruleName} Rule #{rule_id}</Link>);
+      return {
+        id: rule_id,
+        link: invalidHighlightLink
+      }
+    });
+
+    const dataTableFields = [
+      { name: "Invalid Highlights", attribute:"link", width: "100%" }
```

Comment:
based off seeing that `[Object object]` tooltip in the second screenshot and the fact that all the values will be short strings like "Rule #344", you can add `noTooltip: true` to this hash to remove that